rubi/network_config/optimism_goerli/network.yaml

```diff
@@ -15,3 +15,4 @@ token_addresses:
  WETH: "0x4200000000000000000000000000000000000006"
  OP: "0xCeE7148028Ff1B08163343794E85883174a61393"
  USDC: "0xe432f229521eE954f80C83257485405E3d848d17"
+ USDT: "0xd70734ba8101ec28b38ab15e30dc9b60e3c6f433"
```

Comment:
confirmed, `USDT` address supported by the app. can't wait till we have a token list, will be a programmatic way to approach token validation. can even be a part of the CI pipeline